aws/image.yaml

```diff
@@ -5,7 +5,7 @@ env:
     DOCKER_USERNAME: docker-hub-read-only:DOCKER_USERNAME
   variables:
     AWS_REGION: us-east-1
-    GHC_VERSION: 9.2.4
+    GHC_VERSION: 9.2.5
```

Comment:
You'll need to set this in the GitHub workflow:

https://github.com/acilearning/docker-haskell/blob/08f0de6f7cf16cb1707e97ce43370b0649faf1fd/.github/workflows/workflow.yaml#L10-L13

The value here is just the default.